DEVELOPERS_GUIDE.md

```diff
@@ -261,13 +263,13 @@ To run the server, navigate to the `server` folder and execute the following com
 
 ```
 $ cd server
-$ yarn run build -- --env=YOUR_ENV_NAME && node dist/index.js
+$ yarn build --env=YOUR_ENV_NAME && node dist/index.js
```

Comment:
`run` is optional here so can be removed. I'm fine with adding it back. The `--` however should be removed. When running it with `yarn@1` it gives this error message

> From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts

Reply:
Yes, good catch!